### PR TITLE
Added null check in IsLastItemInContainerConverter.

### PR DESCRIPTION
**Pull request by grokys**
#503 is a crash that seems to occur here. Can't reproduce locally but there's only so much that can cause an NRE in this method so try to avoid it in future.

### CODE REVIEW

File: src/GitHub.UI/Converters/IsLastItemInContainerConverter.cs [hunk 1]
```diff
@@ -14,8 +14,15 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
...
-		}
+            if (ic != null && ic.Items != null)
+            {
+                return ic.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1;
+            }
+            else
+            {
+                return false;
+            }
+        }
```
Comment (shana) [inline src/GitHub.UI/Converters/IsLastItemInContainerConverter.cs]: Indentation seems heavily screwed up here.

Also, this is probably enough:

```
return ic != null ? ic.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1 : false;
```

If `ic` is not null than `Items` won't be null either (I peeked, it's created in the getter).

File: src/GitHub.UI/Converters/IsLastItemInContainerConverter.cs [hunk 2]
```diff
@@ -6,27 +6,20 @@
...
 
-            if (ic != null && ic.Items != null)
-            {
-                return ic.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1;
-            }
-            else
-            {
-                return false;
-            }
+            return ic?.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1;
```
Comment (shana) [inline src/GitHub.UI/Converters/IsLastItemInContainerConverter.cs]: Won't it still crash on the right-hand side in `ic.Items.Count`?

Comment (grokys) [inline src/GitHub.UI/Converters/IsLastItemInContainerConverter.cs]: D'oh!

**Comment by shana** [comment]
:rocket:

**Pull request merged**